tools/csv-gen/csv-gen.go

```diff
@@ -332,6 +332,7 @@ func main() {
 		relatedImages = addRefRelatedImages(constants.Datagrid73ImageURL16, constants.Datagrid73Component, imageRef, relatedImages)
 		relatedImages = addRefRelatedImages(constants.Broker75ImageURL, constants.BrokerComponent, imageRef, relatedImages)
```

Comment:
will `Broker75` still be used by any of the supported product versions in 7.9? if not, please remove from csv/constants files and regenerate csv.

Reply:
I don't think so, ok, will remove it.

Reply:
updated.